TacTok/count_names.py

```diff
@@ -62,16 +69,23 @@ def count_in_goal(node):
                                 help='The folder for CoqGym')
     arg_parser.add_argument('--output', type=str, default='./names/',
                                 help='The output file')
+    arg_parser.add_argument('--locals', action='store_true', help='Get local names instead of global names')
     args = arg_parser.parse_args()
     print(args)
+    
+    get_locals = args.locals
 
     iter_proofs(args.data_root, count, include_synthetic=False, show_progress=True)
 
     dirname = args.output
     if not os.path.exists(dirname):
         os.makedirs(dirname)
 
-    names_file = open(os.path.join(dirname, 'names.pickle'), 'wb')
+    filename = 'names.pickle'
+    if get_locals: # I apologize to the functional programming gods, but Python is so confusing
+        filename = 'locals.pickle'
```

Comment:
I think what you want here is `filename = 'locals.pickle' if get_locals else 'names.pickle'`.

Reply:
Yeah but I hate that syntax. I'm just frustrated at the lack of genuine if expressions